site/content/en/docs/concepts/multikueue.md

```diff
@@ -76,6 +76,21 @@ The supported version of the Kubeflow Training Operator is v1.7.0, or a newer ve
 The Management cluster should only install the CRDs and not the package itself. 
 On the other hand, the Worker cluster should install the full kubeflow operator.
 
+## Plain Pods
+
+MultiKueue supports the remote creation and management of Plain Pods.
+
+## Deployments
+
+MultiKueue supports the remote creation and management of Deployment replica Pods.
+
+Known Limitations:
+- When creating a Deployment in environments with more than 1 worker cluster it is possible that replicas are scheduled in different clusters.
```

Comment:
IIUC, this is a common MultiKueue issue regardless of integration types.
Shouldn't we put this in top of this page?
@mimowo Any thoughts?

Reply:
Which integrations in particular do you know? 
Anything with a CR such as RayCluster, KubeFlow should have their workloads copied to a cluster but their pods will run in that cluster. 

The limitation here if I haven't explained it right is that deployment replicas can appear on any cluster e.g. deployment created with 4 replicas, 3 can appear on cluster 1 and 1 can appear on cluster 2. 
We discuss this mode of deployment creation more thoroughly in #4103.

Reply:
Ah, sorry. My understanding does not seem to be correct. I was thought that this point indicates preemption.
In the current MultiKueue env, the workload preemption could happen in multiple clusters even though 1 workload preemption in 1 cluster is enough.

But this point describe the situation where the pod workload could be scheduled to multiple worker clusters since we create a Workload for Pod, not for Deployment.